sample/https-client.c

```diff
@@ -474,6 +474,7 @@ main(int argc, char **argv)
 		SSL_CTX_free(ssl_ctx);
 	if (type == HTTP && ssl)
 		SSL_free(ssl);
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
So seems that initialization also deprecated, can you move add `ifdef` for them too?